### DISCUSSION TITLE
Task20-Flashloan

### DISCUSSION THREAD

**Discussion by liberhe** [Dapp-Learning-DAO/Dapp-Learning#633] [Bounty] [answered]
发起方： Dapp-Learning-DAO
 奖励金:   100dai
 要求: 完成aave, uniswap, dydx闪电贷，matic上能部署。
欢迎认领！

**Comment by sansx** [comment, marked as answer]
认领task20，会好好完成的 😊

**Reply by 0x-stan** [reply]
Done by  
@sansx 's PR:
https://github.com/Dapp-Learning-DAO/Dapp-Learning/pull/643

and 

@yingjingyang 's PR:
https://github.com/Dapp-Learning-DAO/Dapp-Learning/pull/707

**Comment by cyl19970726** [comment]
i will finish it soon
